source/upgrade.txt

```diff
@@ -169,6 +175,12 @@ The v4.0 driver introduces the following breaking changes to the ABI:
   library equivalents. For context, see the :ref:`Build System Breaking Changes
   <version-4.0-build-system-breaking>` section.
 
+- Removes the ``mongocxx::stdx`` namespace. Replace the following
+  types in your code:
+
+  - ``mongocxx::stdx::optional<T>``. Instead, use ``bsoncxx::stdx::optional<T>``.
+  - ``mongocxx::stdx::string_view``. Instead, use ``bsoncxx::stdx::string_view``.
+
```

Comment:
```suggestion
```
This one is not an ABI breaking change, as the actual types are unchanged (via type aliasing).